generator/namespaces.py

```diff
@@ -74,15 +80,21 @@ def namespaces(custom_namespaces, generated_sql_uri, app_listings_uri):
         for app in group:
             if app.get("deprecated"):
                 continue
-            if canonical_app_name is None or app.get("app_channel") == "release":
+            is_release = app.get("app_channel") == "release"
+            if canonical_app_name is None or is_release:
                 canonical_app_name = app["canonical_app_name"]
             dataset_id = app["bq_dataset_family"]
-            for view_id in view_definitions[dataset_id]:
+            for view_id, references in view_definitions[dataset_id].items():
                 if view_id in OMIT_VIEWS:
                     continue
                 table = {"table": f"mozdata.{dataset_id}.{view_id}"}
                 if "app_channel" in app:
                     table["channel"] = app["app_channel"]
+                if len(references) == 1 and references[0][-2] == f"{dataset_id}_stable":
+                    # ping views reference a single table in the stable dataset
+                    table["is_ping"] = True
```

Comment:
`is_ping` is a bit confusing to read without context in `namespaces.yaml`. What do you think of doing the inverse, and calling it `is_derived_table`?

Reply:
I don't like `is_derived_table` because I don't think we should actually categorize everything else as "derived" (e.g. we also have `_static`, `_external`, etc)

per slack, let's go with `is_ping_table`